libraries/dl-streamer/docs/source/dev_guide/advanced_install/advanced_install_guide_compilation.rst

```diff
@@ -112,15 +112,40 @@ Clone and build GStreamer:
     ninja -C build
     sudo env PATH=~/python3venv/bin:$PATH meson install -C build/
 
-Step 6: Build OpenCV (ubuntu 22/fedora)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Step 6: Build OpenCV
+^^^^^^^^^^^^^^^^^^^^
 
-Download and build OpenCV:
+NOTE: If you have build and installed different version of OpenCV locally it can cause build error and it is recomended to uninstall it first.
```

Comment:
```suggestion
NOTE: If you have built and installed a different version of OpenCV locally, it can cause build errors. It is recommended to uninstall it first.
```